solutions/authentication/index.asciidoc

```diff
@@ -0,0 +1,54 @@
+= Authentication
```

Comment:
We have to check if a level one headline is working as expected. In some strange cases, the level one headline is used as the page title, and the text after the level headline is put in the meta tags as a description.

Reply:
Please, think of tags that describe this solution.